usr/lib/systemd/system/greenboot-script-runner.service

```diff
@@ -1,15 +1,17 @@
-#  This file is part of Greenboot.
+#  This file is part of greenboot.
 #
-#  Greenboot is free software; you can redistribute it and/or modify it
+#  greenboot is free software; you can redistribute it and/or modify it
 #  under the terms of the GNU Lesser General Public License as published by
 #  the Free Software Foundation; either version 2.1 of the License, or
 #  (at your option) any later version.
 
 [Unit]
-Description=Greenboot Required Scripts Check
-Before=greenboot.service
+Description=greenboot Success Scripts Runner
+Before=greenboot.target
+
+[Install]
 RequiredBy=greenboot.target
 
 [Service]
 Type=oneshot
-ExecStart=/bin/run-parts --list /etc/greenboot/requires | /bin/systemd-cat -p notice
+ExecStart=/usr/libexec/greenboot/greenboot.sh green
```

Comment:
In the current (0.1) this would mean changing the `Wants=greenboot.service` on the target to `Requires`, creating a circular dependency. Is this what we want? It does seem quite nice though to not only enter red state on health check failure but also on green script failure, which this would probably achieve